src/holodeck/agents.py

```diff
@@ -448,16 +492,30 @@ class HandAgent(HolodeckAgent):
     Inherits from :class:`HolodeckAgent`.
     
     """
+    # constants in HandAgent.h in holodeck-engine
+    __MAX_MOVEMENT_METERS = 0.5
+    __MIN_MOVEMENT_METERS = __MAX_MOVEMENT_METERS
+
+    __MAX_TORQUE = 20
+    __MIN_TORQUE = -__MAX_TORQUE
 
     agent_type = "HandAgent"
 
     @property
     def control_schemes(self):
-        return [("[Raw Bone Torques] * 23", ContinuousActionSpace([23])),
+        raw_min = [self.__MIN_TORQUE for _ in range(23)]
```

Comment:
23 should be a const variable, and maybe 26.

Reply:
Done